### PR TITLE
Add PFT Mask.

### LINKED ISSUE #412: Add PFT to api using masks.

### PR DESCRIPTION
**Pull request by 36000**
Closes #412 

- [x] Add tests

### CODE REVIEW

File: AFQ/tests/test_api.py [hunk 1]
```diff
@@ -292,6 +292,55 @@ def test_AFQ_reco():
...
+        'vistasoft',
+        'sub-01',
+        'ses-01',
+        'dwi')
+    img = nib.load(op.join(sub_path, 'sub-01_ses-01_dwi.nii.gz'))
+    pve_wm_data = nib.Nifti1Image(np.ones(img.shape[:3]), img.affine)
+    pve_gm_data = nib.Nifti1Image(np.zeros(img.shape[:3]), img.affine)
+    pve_csf_data = nib.Nifti1Image(np.zeros(img.shape[:3]), img.affine)
+
+    nib.save(pve_wm_data,
```
Comment (arokem) [inline AFQ/tests/test_api.py:320]: There is PVE data for the stanford data set that we can use in this test: 

```
from dipy.data import get_fnames
f_pve_csf, f_pve_gm, f_pve_wm = get_fnames('stanford_pve_maps')
```

**Comment by arokem** [comment]
Looks like this PR passed before marking as nightly: https://github.com/yeatmanlab/pyAFQ/runs/1135396809?check_suite_focus=true. Is it ready to merge, or do we need to do anything else here?

**Comment by arokem** [comment]
Also, not necessarily something to do here, but do we have any way to tell the AFQ object to seed for tracking in the GM/WM interface, based on `pve_wm > 0.1 and pve_gm > 0.1`?

**Comment by 36000** [comment]
> Looks like this PR passed before marking as nightly: https://github.com/yeatmanlab/pyAFQ/runs/1135396809?check_suite_focus=true. Is it ready to merge, or do we need to do anything else here?

Its currently running again with the changes you suggested (use segmentations from the stanford data set)

**Comment by 36000** [comment]
> Also, not necessarily something to do here, but do we have any way to tell the AFQ object to seed for tracking in the GM/WM interface, based on `pve_wm > 0.1 and pve_gm > 0.1`?

Yes.
```
seed_mask = CombinedMaskFile([
    ThresholdedMaskFile('WMprobseg',  lower_bound=0.1),
    ThresholdedMaskFile('GMprobseg',  lower_bound=0.1)])
```

**Pull request merged**